ddpui/ddpdbt/dbt_service.py

```
@@ -253,3 +323,450 @@ def check_repo_exists(gitrepo_url: str, gitrepo_access_token: str | None) -> boo
+                k: v
+                for k, v in nodes.items()
+                if v.get("resource_type") == "model" and v.get("package_name") == project_name
+            }
+            logger.info(
+                f"Processing {len(model_nodes)} models from manifest (filtered to project: {project_name})"
+            )
+
+            for node_id, node_data in model_nodes.items():
+                # Extract model information
+                model_name = node_data.get("name", "")
+                database = node_data.get("database", "")
```

Ruff (0.14.10)

Local variable `database` is assigned to but never used

Remove assignment to unused variable `database`

(F841)